src/SprykerMiddleware/Shared/Process/Transfer/process.transfer.xml

```diff
@@ -14,4 +15,8 @@
         <property name="parseAsArray" type="bool"/>
     </transfer>
 
+    <transfer name="AggregatorSettings">
+        <property name="sequenceSizeRelease" type="int"/>
```

Comment:
As discussed, we do not always need this field, but we have to check whether we can provide an empty transfer definition.